### ISSUE TITLE
Can't copy or open test file from the mounted filesystem created in cryptofs 

### ISSUE THREAD

**Issue by ihsanhaikalz** [cryptomator/cryptofs#3]
I followed the example mentioned in the readme that will generate a test file with "test" written in it using cryptofs. I tried to mount it using cryptomator and it works but when I tried to copy the test file to normal directory, eg. C://testfolder, it said "Can't read from the source file or disk." Also i created a new .txt file to confirm it and when I tried to open it with Notepad it gave me error "A device attached to the system is not functioning". Is it a bug or is it supposedly like this?

**Comment by markuskreusch** [comment]
There are some known problems with the webdav share in windows which might apply here. I guess the problem is not cryptofs / the crypto code in Cryptomator but something related to webdav.

Can you try to mount the Cryptomator webdav share using a third party webdav client like Cyberduck?
You can copy the webdav URL from the Cryptomator UI when you click on the arrow next to the lock button.

If it works well with Cyberduck (e.g. you can read and write files) cryptofs should be fine.

If it doen't please start Cryptomator and enable the debug mode in the settings. Afterwards restart Cryptomator and reproduce the problem. Then attach the logfile here.

**Comment by ihsanhaikalz** [comment]
I have tried using Cyberduck but it gave me error 500 Server Error. Also I thought something is wrong with Cryptofs so I mounted again the vault and copy real file into the vault and lock it. Again I tried to mount again and copy the real file into some directory and open it and it works, so I think the problem should be with the writing on the filesystem while it is still not mounted and decrypted. Attached is the logfile that I converted it into pdf type
[cryptomator.pdf](https://github.com/cryptomator/cryptofs/files/633323/cryptomator.pdf)
